x-pack/plugin/downsample/qa/rest/build.gradle

```diff
@@ -12,6 +12,15 @@ apply plugin: 'elasticsearch.internal-test-artifact'
 dependencies {
   testImplementation project(path: ':test:test-clusters')
   yamlRestTestImplementation project(path: xpackModule('rollup'))
+
+  clusterModules project(path: xpackModule('downsample'))
```

Comment:
FWIW, `project(path: xpackModule('downsample'))` can be simplified to just `project(xpackModule('downsample'))`. In fact I think the former syntax is going to be deprecated by Gradle at some point.